### ISSUE TITLE
Random test failure in Test_SwshFiltering

### ISSUE THREAD

**Issue by kidder** [sxs-collaboration/spectre#1733]
# Bug reports:

```
/work/spectre/tests/Unit/NumericalAlgorithms/Spectral/Test_SwshFiltering.cpp:125
...............................................................................
/work/spectre/tests/Unit/TestingFramework.hpp:162: FAILED:
  CHECK( a == appx(b) )
with expansion:
  -0.0134639885 == Approx( -0.0134639885 )
with messages:
  Seed is: 1088894636 from /work/spectre/tests/Unit/NumericalAlgorithms/
  Spectral/Test_SwshFiltering.cpp:31
  l_max := 7
  Spin := 2
...
 a: -1.346398849102956041e-02
  b: -1.346398849105209794e-02
```

### Expected behavior:

Test passes

### Current behavior:

Test fails

### Environment:

https://travis-ci.org/sxs-collaboration/spectre/jobs/600936029?utm_medium=notification&utm_source=github_status

**Comment by kidder** [comment]
Another failure 

https://github.com/sxs-collaboration/spectre/pull/1764/checks?check_run_id=318788657

**Comment by kidder** [comment]
@moxcodes please fix, thanks!

**Comment by nilsdeppe** [comment]
```
2019-11-25T07:03:38.9083775Z with expansion:
2019-11-25T07:03:38.9083862Z   0.0332435734 == Approx( 0.0332435734 )
2019-11-25T07:03:38.9083967Z with messages:
2019-11-25T07:03:38.9084076Z   Seed is: 67025610 from /__w/spectre/spectre/tests/Unit/NumericalAlgorithms/
2019-11-25T07:03:38.9084195Z   Spectral/Test_SwshFiltering.cpp:31
2019-11-25T07:03:38.9084298Z   l_max := 7
2019-11-25T07:03:38.9084384Z   Spin := 2
```

**Comment by moxcodes** [comment]
I'll fix this today.

**Comment by moxcodes** [comment]
closing because it was fixed by #1822
